spec/features/create_media_audio_spec.rb

```diff
@@ -0,0 +1,106 @@
+# frozen_string_literal: true
+
+require 'fileutils'
+require 'random_word'
```

Comment:
This is not necessary, since it's in the Gemfile

Reply:
is this new - that we don't need require??  or something special for random_word?

Reply:
Anything in the Gemfile gets required in a Rails app unless you put `require: false`.  This happens when you call `Bundler.require` (https://github.com/sul-dlss/pre-assembly/blob/e5091a1fbb4e385f69c49d25a531a63c118c401f/config/application.rb#L10)